src/styles/blocks/card.scss

```
@@ -0,0 +1,81 @@
+
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: $cardSize;
+  height: 406px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  &__img-wrapper {
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__img {
+    height: 100%;
+    width: 100%;
+    object-fit: contain;
+  }
+
+  &__product-name {   //*   это навзание моего товара
```

It's a redundant comment. Delete this
```suggestion
  &__product-name {
```